qa/common/gen_qa_identity_models.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Can you also ask Hemant about why he didn't already create an identity model for PyTorch. Perhaps he thought these models were only for "zero-sized tensor" testing and pytorch doesn't support that so he didn't create it. But he needs to make sure that we have pytorch testing in all the other places that use the identity model.

Reply:
On it!